signingscript/src/signingscript/sign.py

```diff
@@ -927,6 +939,15 @@ def make_signing_req(input_file, fmt, keyid=None, extension_id=None):
     return sign_req
 
 
+@time_function
+def make_files_signing_req(input_files, fmt, keyid=None, options=None):
+    """Make a signing request to pass to the /sign/files endpoint in autograph."""
+    signing_req = {"keyid": keyid, "options": options, "files": []}
+    for input_file in input_files:
+        signing_req["files"].append({"name": input_file.name, "content": b64encode(input_file.read())})
+    return signing_req
```

Comment:
Maybe we can just merge this with `make_signing_req` now?
```
if  len(input_files) > 1:
    sign_req = {"input": input_files}
else:
    sign_req = {"files": []}
    for input_file in input_files:
        ...

Reply:
Though I'm not sure what `options` is for.. If that's only something used by the files endpoint, then maybe keeping them separate is better...

Reply:
Files does have an option field I combined these